### PR TITLE
GDAL Vector misc - Tables for alg parameters

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst [hunk 1]
```diff
@@ -62,184 +103,281 @@ This algorithm is derived from the `GDAL ogr2ogr utility <https://gdal.org/ogr2o
...
+
+        Default: ''
+      - Defines a distance tolerance for the simplification of the
+        vector geometries to be imported.
+        By default no simplification there is no simplification.
+   *  - **Maximum distance between 2 nodes (densification)**
+
+        Optional
+      - ``SEGMENTIZE``
+      - [string]
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst]: Agree.

File: source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst [hunk 2]
```diff
@@ -62,184 +103,281 @@ This algorithm is derived from the `GDAL ogr2ogr utility <https://gdal.org/ogr2o
...
+        table.
+        By default a new table will be created.
+   *  - **Do not launder columns/table names**
+
+        Optional
+      - ``LAUNDER``
+      - [boolean]
+
+        Default: False
+      - With this option you can prevent processing from
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst]: I have updated the description.

File: source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst [hunk 3]
```diff
@@ -62,184 +103,281 @@ This algorithm is derived from the `GDAL ogr2ogr utility <https://gdal.org/ogr2o
...
+        converting column names to lowercase, from removing
+        spaces and other invalid characters.
+   *  - **Do not create Spatial Index**
+
+        Optional
+      - ``INDEX``
+      - [boolean]
+
+        Default: False
+      - Prevents that a spatial index in the output table will be created
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst]: Should be fixed.

File: source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst [hunk 4]
```diff
@@ -62,184 +106,279 @@ This algorithm is derived from the `GDAL ogr2ogr utility <https://gdal.org/ogr2o
...
-``Additional creation options`` [string]
-  Optional
+Outputs
+.......
 
-  <put parameter description here>
+This algorithm has no output.
 
 
 .. _importvectorintopostgisdatabasenewconnection:
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst]:
```suggestion
.. _gdalimportvectorintopostgisdatabasenewconnection:
```

File: source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst [hunk 5]
```diff
@@ -13,40 +13,84 @@ Vector miscellaneous
...
+     - Name
+     - Type
+     - Description
+   * - **SQL result**
+     - ``OUTPUT``
+     - [vector: any]
+     - Vector layer created by the query
+
 
 .. _importvectorintopostgisdatabaseavailableconnection:
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/vectormiscellaneous.rst]:
```suggestion
.. _gdalimportvectorintopostgisdatabaseavailableconnection:
```

**Comment by havatv** [comment]
Thanks for the thorough review, @DelazJ!

**Pull request merged**